dt-assets/parts/modals/modal-help.php

```diff
@@ -12,17 +12,17 @@
         ?>
         <!--    Contact Details Tile  -->
         <div class="help-section" id="contact-details-help-text" style="display: none">
-            <h3><?php echo esc_html_x( "Contact Details", 'disciple_tools' ) ?></h3>
+            <h3><?php esc_html_e( "Contact Details", 'disciple_tools' ) ?></h3>
             <p><?php echo esc_html_x( "This is the area where you can view and edit the contact details for this contact.", 'Optional Documentation', 'disciple_tools' ) ?></p>
             <ul>
-                <li><strong><?php echo esc_html_x( "Contact Name", 'disciple_tools' ) ?></strong></li>
+                <li><strong><?php echo esc_html( "Contact Name", 'disciple_tools' ) ?></strong></li>
```

Comment:
Please fix this one and all the others like it.

Reply:
`<?php esc_html_e( "Contact Name", 'disciple_tools' ) ?>`